Altis_Life.Altis/stringtable.xml

```diff
@@ -4560,7 +4560,7 @@
             <Czech>Jste utekl z vězení, si stále zachovávají své předchozí zločiny a nyní mají počet uniknout vězení.</Czech>
             <Spanish>Te has escapado de la cárcel, retienes tus crimenes anteriores más un crimen de escapado de la cárcel.</Spanish>
             <Russian></Russian>
-            <German>Du bist aus dem Gefängnis ausgebrochen, die Polizei sucht bereits wegen deiner früheren Verbrechen und Gefängnisausbruch nach dir.</German>
+            <German>Du bist aus dem Gefängnis ausgebrochen! Die Polizei sucht dich jetzt wegen deiner früheren Verbrechen und dem Gefängnisausbruch nach dir.</German>
```

Comment:
Du bist aus dem Gefängnis ausgebrochen! Die Polizei sucht dich jetzt wegen deiner früheren Verbrechen und dem Gefängnisausbruch.
without **nach dir** at the end